### PR TITLE
OBPIH-2825 Redirect to incoming stock movement list from requisition …

### PR DESCRIPTION
**Pull request by DominikaHatala**
…workflow

### CODE REVIEW

File: src/js/components/stock-movement-wizard/request/AddItemsPage.jsx [hunk 1]
```diff
@@ -1072,7 +1072,7 @@ class AddItemsPage extends Component {
     if (this.state.values.statusCode === 'CREATED') {
       return apiClient.post(url, payload)
         .then(() => {
-          window.location = `/openboxes/stockMovement/list?type=REQUEST&movementNumber=${movementNumber}&submitted=true`;
+          window.location = `/openboxes/stockMovement/list??direction=INBOUND&movementNumber=${movementNumber}&submitted=true`;
```
Comment (jmiranda) [inline src/js/components/stock-movement-wizard/request/AddItemsPage.jsx:1075]: Remove extra ?

Comment (DominikaHatala) [reply src/js/components/stock-movement-wizard/request/AddItemsPage.jsx:1075]: @jmiranda I wanted to remove it but then I realized that once user submits request there is message displayed "Thank you for submitting your request. You can check the status of your request using stock movement number ...". So we need both params - movement number and submitted boolean.

Comment (jmiranda) [reply src/js/components/stock-movement-wizard/request/AddItemsPage.jsx:1075]: I assume you misinterpreted what I said. There’s no way you need two question marks (??) in the URL.

Comment (DominikaHatala) [reply src/js/components/stock-movement-wizard/request/AddItemsPage.jsx:1075]: @jmiranda Ohhh... Yes, I totally misinterpreted. Of course, you're right!

Comment (DominikaHatala) [reply src/js/components/stock-movement-wizard/request/AddItemsPage.jsx]: @jmiranda Updated PR.

**Pull request merged**